actionlib_tools/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>actionlib_tools</name>
+  <version>0.1.0</version>
+  <description>The actionlib_tools package</description>
```

Comment:
Please fill this out, in particular calling out that the tools have GUI dependencies.

Reply:
Updated in a45cdf8